content/_data/changelogs/weekly.yml

```diff
@@ -7957,6 +7957,97 @@
   # pull: 4777 (PR title: Use the 'agent' term more often)
   # pull: 4778 (PR title: Fixed some style errors in less files to reduce output during build)
 
+- version: "2.242"
+  date: 2020-06-23
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4802
+    issue: 62622
+    authors:
+    - daniel-beck
+    message: |-
+      * Do not log exceptions when browsing to the plugin manager while private source plugins are installed (regression in 2.240).
+  - type: bug
+    category: bug
```

Comment:
Any reason why we put bugs before rfes? This is not how the Changelog Generator rules are documented or how the previous changelogs are written:

* major bug
* major rfe
* rfe
* bug
* localization
* developer
* internal

Reply:
My mistake.  Thanks for making the fix